app/src/interfaces/many-to-many/use-selection.ts

```diff
@@ -47,7 +47,7 @@ export default function useSelection(
 		const selection = newSelection.reduce((acc, item) => {
 			if (selectedPrimaryKeys.value.includes(item) === false) acc.push({ [junctionField]: item });
 			return acc;
-		}, new Array());
+		}, [] as Record<string, any>[]);
```

Comment:
This change was necessary, TS was complaining about type of array being `never[]` which does not accept `item`.